css-basics/css-structure_custom-properties/css/styles.css

```diff
@@ -4,19 +4,25 @@
   box-sizing: border-box;
 }
 
+:root {
+  --color-black-light: rgb(48 48 48 / 100%);
```

Comment:
I know that black & white occur the same amount of times but maybe it would still be better to rename them in a way that is not the color? I think Sven talked about how if "the client" wants to change let's say "color-light-black" into yellow not, your code would say light black everywhere where it would then be yellow. Get the idea?